docs/modules/pe.rst

```diff
@@ -1017,6 +1025,120 @@ Reference
 
     *Example:  pe.imports(/kernel32\.dll/i, /(Read|Write)ProcessMemory/) == 2*
 
+.. c:function:: delayed_imports(dll_name, function_name)
```

Comment:
What if instead of adding `delayed_imports` and `any_imports` we simply implement another version of `imports` that receives an extra argument indicating if you are interested in delayed imports only, standard imports only, or both. The extra argument could be an integer where one bit, means "include delayed imports" and another bit means "include standard imports", by xoring both values you can indicate that you want both.

The current `imports` function can be implemented by calling this new function.

Reply:
I will implement it your way. At first, I implemented it this way, because it  seemed "cleaner" for me. But now after I read your comment, I agree that it will be better to use binary flags parameter instead of functions. This way there will be one function instead of three.

Examples:

`pe.imports(lib, fun, pe.STANDARD_IMPORT )`
`pe.imports(lib, fun, pe.DELAY_IMPORT)`
`pe.imports(lib, fun, pe.STANDARD_IMPORT | pe.DELAY_IMPORT)`

Reply:
Exactly, that's the idea.

Reply:
Refactored imports functions

Same as example except order of parameters. Flag specifying which import to search is first. If flag was last it will create multiple overloads with same parameters (`lib, ordinal` vs `lib, flag`) 

Example:

- `pe.imports(, pe.STANDARD_IMPORT, lib, fun)`